test/jest.config.js

```diff
@@ -20,8 +20,8 @@ module.exports = {
   roots: ['<rootDir>'],
   coverageDirectory: './coverage',
   moduleNameMapper: {
-    // "\\.(css|less|scss)$": "<rootDir>/test/mocks/styleMock.ts",
-    // "^ui/(.*)": "<rootDir>/../../src/legacy/ui/public/$1/",
+    "\\.(css|less|sass|scss)$": "<rootDir>/__mocks__/styleMock.js",
+    "\\.(gif|ttf|eot|svg)$": "<rootDir>/__mocks__/fileMock.js"
```

Comment:
What are these two lines about?

Reply:
To solve the errors when importing `_index.scss`.
https://github.com/facebook/jest/issues/3094